projects/jump-search/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Jump Search in Python
```

Comment:
```suggestion
title: Jump Search in Every Language
```

Reply:
Actually I had put it up in python only, thats why I named it jump search in python.

Reply:
Oh I understand, but the documentation would be for all languages. In other words, you would write this, and other people could replicate it.